tests/test_ephys_job.py

```diff
@@ -1116,6 +1119,65 @@ def test_run_job(
         mock_compress_raw_data.assert_called_once()
 
 
+class TestCheckTimeAlignment(unittest.TestCase):
+    """Tests time alignment check"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Setup basic job settings and job that can be used across tests."""
+        basic_job_settings_raise = EphysJobSettings(
+            input_source=FAKE_HARP_DIR,
+            output_directory=Path("output_dir_align"),
+            compress_job_save_kwargs={"n_jobs": 1},
+        )
+        cls.basic_job_settings_raise = basic_job_settings_raise
+        cls.basic_job_raise = EphysCompressionJob(
+            job_settings=basic_job_settings_raise
+        )
+
+        basic_job_settings_warn = EphysJobSettings(
+            input_source=FAKE_HARP_DIR,
+            output_directory=Path("output_dir_align"),
+            compress_job_save_kwargs={"n_jobs": 1},
+            check_timestamps=False,
+        )
+        cls.basic_job_settings_warn = basic_job_settings_warn
+        cls.basic_job_warn = EphysCompressionJob(
+            job_settings=basic_job_settings_warn
+        )
+
+    @classmethod
+    def tearDownClass(cls):
+        """Cleanup any resources used in tests"""
+        shutil.rmtree(Path("output_dir_align"))
```

Comment:
We should avoid reading and writing files in the unit tests.

Reply:
@jtyoung84 done :)